### ISSUE TITLE
bitcoinrpc provider broken in 0.80

### ISSUE THREAD

**Issue by skoroneos** [richardkiss/pycoin#258]
Greetings,

after updating (with pip) to 0.80 Tx fails to connect to the local (bitcoin core) full node to retrieve transactions.
The older version (0.7x i think) was working fine.
As far i can tell from a quick debug session, it opens the connection but never sends any follow up rpc commands (i.e getrawtransaction) and since there is no local cache it returns "tx: error: can't find Tx with id  XXXXXXX"

**Comment by richardkiss** [comment]
I did find a problem, fixed in `develop` in 35a66958bbf828b246993817fe2c5d06ce96c691. Looks like it can be easily backported to 0.80.

**Comment by richardkiss** [comment]
Fixed in master in f681c997595097571e45a58adb93cd5d5b39ca53.

**Comment by richardkiss** [comment]
Sorry, turns out that fix was just a comment. I'm reopening.

**Comment by darklow** [comment]
I was able to use my own bitcoin testnet node through rpc using following way:
```
export PYCOIN_CACHE_DIR=~/.pycoin_cache
export PYCOIN_XTN_PROVIDERS=bitcoinrpc://user:pass@host:18332
```
At first I was having same issue and error as @skoroneos because of two mistakes I did:
1. I was specifying `PYCOIN_BTC_PROVIDERS` instead of `PYCOIN_XNT_PROVIDERS` and using `tx -nXTN [txid]`
2. Even I fixed No1 I was still getting error `tx: error: can't find Tx with id` unless I specified `PYCOIN_CACHE_DIR`. Once I did that all started working. Looks like this issue is fixed in `0.9` and `PYCOIN_CACHE_DIR` is optional.

So everything works now, at least to me on both `0.8` and also `0.9-dev` versions once both env params are set. @skoroneos Hope this helps to you as well or try `pip install -e git://github.com/richardkiss/pycoin.git@develop#egg=pycoin`
